### PR TITLE
[ADD] account_invoice_report_utilities: New field "Invoice printing n…

### PR DESCRIPTION
**Pull request by alfredoavanzosc**
…ote" in account tax, and in account fiscal position.

### CODE REVIEW

**Comment by alfredoavanzosc** [comment]
@avanzosc/developers ... review please

File: account_invoice_report_utilities/README.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================================
+Account invoice report utilities
+================================
+
+* New fiedl "Invoice printing note" in account tax, and in account fiscal
```
Comment (oihane) [inline account_invoice_report_utilities/README.rst]: misstyped

File: account_invoice_report_utilities/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Copyright 2019 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from . import models
```
Comment (oihane) [inline account_invoice_report_utilities/__init__.py:2]: Si solo se importa no se pone copyright

**Comment by alfredoavanzosc** [comment]
Realizadas las mejoras solicitadas por @oihane. Eskerrik asko.

**Review by Daniel-CA — APPROVED** [review]

**Review by mikelarre — APPROVED** [review]

**Pull request merged**